addons/pos_cache/static/src/js/pos_cache.js

```diff
@@ -7,20 +7,14 @@ const Registries = require('point_of_sale.Registries');
 
 const PosCachePosGlobalState = (PosGlobalState) => class PosCachePosGlobalState extends PosGlobalState {
     async _getTotalProductsCount() {
```

Comment:
do we need the `async` here since we're not awaiting ?

Reply:
async is not needed